bitnami/jaeger/templates/query/deployment.yaml

```diff
@@ -136,37 +146,44 @@ spec:
           {{- else if .Values.query.startupProbe.enabled }}
           startupProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.query.startupProbe "enabled") "context" $) | nindent 12 }}
             httpGet:
-              path: /
-              port: admin
+              path: /status
+              port: healthcheck
           {{- end }}
           {{- if .Values.query.customLivenessProbe }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.query.customLivenessProbe "context" $) | nindent 12 }}
           {{- else if .Values.query.livenessProbe.enabled }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.query.livenessProbe "enabled") "context" $) | nindent 12 }}
-            tcpSocket:
-              port: admin
+            httpGet:
+              path: /status
+              port: healthcheck
           {{- end }}
           {{- if .Values.query.customReadinessProbe }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.query.customReadinessProbe "context" $) | nindent 12 }}
           {{- else if .Values.query.readinessProbe.enabled }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.query.readinessProbe "enabled") "context" $) | nindent 12 }}
             httpGet:
-              path: /
-              port: admin
+              path: /status
+              port: healthcheck
           {{- end }}
           {{- end }}
           {{- if .Values.query.resources }}
           resources: {{- toYaml .Values.query.resources | nindent 12 }}
           {{- else if ne .Values.query.resourcesPreset "none" }}
           resources: {{- include "common.resources.preset" (dict "type" .Values.query.resourcesPreset) | nindent 12 }}
           {{- end }}
-          {{- if .Values.query.extraVolumeMounts }}
-          volumeMounts: {{- include "common.tplvalues.render" ( dict "value" .Values.query.extraVolumeMounts "context" $) | nindent 12 }}
-          {{- end }}
+          volumeMounts:
+            - name: configuration
+              mountPath: /opt/bitnami/jaeger/conf
+            {{- if .Values.query.extraVolumeMounts }}
+            {{- include "common.tplvalues.render" ( dict "value" .Values.query.extraVolumeMounts "context" $) | nindent 12 }}
+            {{- end }}
         {{- if .Values.query.sidecars }}
         {{- include "common.tplvalues.render" ( dict "value" .Values.query.sidecars "context" $) | nindent 8 }}
         {{- end }}
       volumes:
+        - name: configuration
+          configMap:
+            name: {{ include "jaeger.query.configMapName"  . }}
```

Comment:
The annotations should include the configmap checksum to ensure changes trigger a rolling update.